mantidproject/mantid
# Add Linux desktop icon for workbench

**Description of work.**

Adds a desktop launcher and icon for the workbench to the Linux desktops.

This also introduces a suffixed launcher and icon for the nightly/unstable builds to allow them to be started without using the terminal.

**To test:**

Build 2 packages on Linux using the cmake options:
1. `-DCPACK_PACKAGE_SUFFIX="" -DCMAKE_INSTALL_PREFIX=/opt/Mantid`
2. `-DCPACK_PACKAGE_SUFFIX="nightly" -DCMAKE_INSTALL_PREFIX=/opt/mantidnightly`

Check you can install both side-by-side. On Unity I then see 4 icons; MantidPlot and MantidWorkbench for each install type:

![linux_icons_suffixed](https://user-images.githubusercontent.com/733773/56224332-5d9d5200-6067-11e9-87cb-0884c3473219.png)

Check starting the applications starts the correct program.

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
